### PR TITLE
Added Min-Max 0->1 normalization

### PR DESCRIPTION
**Pull request by borondics**
Implements #432 

I have a feeling that this is not very elegant, but it seems to work just fine. @markotoplak, what do you think?

### CODE REVIEW

**Comment by stuart-cls** [comment]
Ha I also thought Friday was a good time to do this :) Mine handles Nans but is other wise exactly the same https://github.com/stuart-cls/orange-spectroscopy/tree/minmax-normalization

**Comment by stuart-cls** [comment]
I'm just going to open a PR to see if mine passes Tests, mostly for personal curiosity :)

File: orangecontrib/spectroscopy/preprocess/__init__.py [hunk 1]
```diff
@@ -312,6 +312,12 @@ def transformed(self, data):
                                   limits=[[self.lower, self.upper]])(data)
             data.X /= norm_data.X
             replace_infs(data.X)
+        elif self.method == Normalize.MinMax:
+            min_data = np.min(data.X, axis=1)
+            max_data = np.max(data.X, axis=1)
+            norm_data = (data.X.T - min_data) / (max_data - min_data)
```
Comment (stuart-cls) [inline orangecontrib/spectroscopy/preprocess/__init__.py:318]: Why all the transposing?

Comment (borondics) [reply orangecontrib/spectroscopy/preprocess/__init__.py:318]: Otherwise, the sizes are not compatible.

Comment (stuart-cls) [reply orangecontrib/spectroscopy/preprocess/__init__.py:318]: Oh I see. You can also pass `keepdims=True` to min/max for the same effect.

**Comment by markotoplak** [comment]
Thank you both! :)

**Pull request closed without merging**